<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="2"
            tests="1">

  <testsuite name="SearchDressFail"
    errors="0" failures="2" hostname="" id="" package="SearchDressFail" skipped="0"
    tests="1" time="12.12" timestamp="">
  
    <testcase name="Verify product names contain &#34;dress&#34;" classname="SearchDressFail" time="2.137" assertions="9">

    

    

    

    

    

    

      <failure message="Failed [ok]: (Product name contains &#34;dress&#34; for product 6) - expected &#34;true&#34; but got: &#34;false&#34; (0ms)">    at NightwatchAPI.&lt;anonymous&gt; (C:\Users\Maria\OneDrive - Faculty of Engineering Ain Shams University\Desktop\SDET_Task_Mariam_Waleed\UI_Testing\tests\SearchDressFail.js:36:30)</failure>

      <failure message="Failed [ok]: (Product name contains &#34;dress&#34; for product 7) - expected &#34;true&#34; but got: &#34;false&#34; (0ms)">    at NightwatchAPI.&lt;anonymous&gt; (C:\Users\Maria\OneDrive - Faculty of Engineering Ain Shams University\Desktop\SDET_Task_Mariam_Waleed\UI_Testing\tests\SearchDressFail.js:36:30)</failure>

    
    </testcase>
  

  

    

  
  </testsuite>
</testsuites>
